org.getmonero.Monero.yaml

```diff
@@ -6,6 +6,7 @@ finish-args:
   - --share=network
   - --share=ipc
   - --socket=x11
+  - --socket=wayland
```

Comment:
Monero-gui does not work very well with wayland, but since someone still needs it, I will accept it.